.github/workflows/ci.yml

```diff
@@ -150,19 +164,12 @@ jobs:
       - uses: actions/checkout@v2
         name: Checkout code
 
-      - name: Pull snuba CI images
-        run: |
-          docker pull ghcr.io/getsentry/snuba-ci:${{ github.sha }} || \
-            docker pull ghcr.io/getsentry/snuba-ci:${{ needs.snuba-image.outputs.branch }} || \
-            docker pull ghcr.io/getsentry/snuba-ci:latest || true
-
-      - name: Build snuba docker image for CI
+      - name: Build testing image
```

Comment:
Making it more clear that this is different than the Snuba image.

Reply:
isn't this a local dev image?